ui/desktop/app/services/origin.js

```diff
@@ -66,7 +66,8 @@ export default class OriginService extends Service {
       // Silently drop trailing slashes if present.
       // This is important since API paths that will be appended to the origin
       // will include preceeding slashes already.
-      origin = origin.replace(/\/*$/, '');
+      // Trim whitespaces around origin.
+      origin = origin.replace(/\/*$/, '').trim();
```

Comment:
We may want to trim first, before removing slashes.  It's common for whitespace to appear at the beginning or end of the string.  The regex is looking for slashes at the immediate end, meaning that if there is whitespace at the end `foobar//    `, the slashes would remain and then the whitespace removed, resulting in `foobar//`.  The result we want is `foobar`.

Reply:
👍 Good catch. Updated